authorizerd.go

```diff
@@ -304,7 +368,20 @@ func (a *authorizer) verify(ctx context.Context, m mode, tok, act, res string) e
 	return nil
 }
 
-// VerifyAccessToken verifies the access token for specific resource and return and verification error.
+// VerifyAccessToken verifies the HTTP request on the specific (action, resource) pair and returns verification error if unauthorized.
+func (a *authorizer) Verify(r *http.Request, act, res string) error {
+	for _, verifier := range a.verifiers {
+		// OR logic on multiple credentials
```

Comment:
If we want to allow multiple credentials, I think it needs to be in OR, but I'll fix that here.